docs/source/library/simple_simulation.rst

```diff
@@ -0,0 +1,6 @@
+simple_simulation
+========================
+
+Function representing the simple simulation of a circuit.
```

Comment:
Whatever "simple" should mean in that case. This point already holds for the original function name. The SyReC library has no "complex" simulation alternative.

Reply:
done 👍🏻